src/client/components/household-information/AnnualIncomeSection.jsx

```diff
@@ -105,22 +116,22 @@ class AnnualIncomeSection extends React.Component {
       spouseIncomeInput = (
         <div className="input-section">
           <h6>Spouse</h6>
-          <ErrorableTextInput
-              errorMessage={getErrorMessage(this.props.data.spouseGrossIncome, message)}
+          <ErrorableTextInput required
```

Comment:
the fields that are required on the spouse and dependent pages are already marked so, so those don't have to be touched